README.md

```diff
@@ -31,6 +31,8 @@ For a development install, clone the repository and then run the following in th
 ```bash
 pip install .
 jlpm build:dev
+jupyter labextension install .
+jupyter serverextension enable jupyterlab_hdf
```

Comment:
Are you sure that you needed those two lines to make the dev install work? 

- `jupyter labextension link .` runs automatically as part of the `jlpm build:dev` command
- you shouldn't need to manually enable the server extension, as that's handled automatically in `setup.py` (by installing the `jupyter-config/jupyter_notebook_config.d/jupyterlab_hdf.json` appropriately)

If the `build:dev` command and the automatic server extension enabling in `setup.py` aren't working as intended, then that's a separate bug I need to fix.

Reply:
That's true, OK I reverted this as well.